frontend/src/hooks/usePickerValues.tsx

```diff
@@ -33,20 +33,20 @@ export const formatPickerOptions = (tableData: IPickerTableData) => {
   });
 };
 
-export const usePickerValues = () => {
+export const usePickerValues = (projectId: number | undefined) => {
```

Comment:
Seems like we're going to have to rethink the whole picker options system pretty soon. We're going to need this kind of id-specific picker for the contract invoices stuff so specifying projectId here won't really work. This is probably the only way to do it for now without making big changes to the picker stuff though.

Reply:
yes we are, i am going to make a ticket for this.